File: mobile/src/main/scala/com/github/shadowsocks/plugin/NoPlugin.scala
package com.github.shadowsocks.plugin

import com.github.shadowsocks.ShadowsocksApplication.app

/**
  * @author Mygod
  */
object NoPlugin extends Plugin {
  override def id: String = ""
  override def label: CharSequence = app.getText(com.github.shadowsocks.R.string.plugin_disabled)
}

File: mobile/src/main/java/com/github/shadowsocks/plugin/NoPlugin.kt
package com.github.shadowsocks.plugin

import com.github.shadowsocks.App.Companion.app

object NoPlugin : Plugin() {
    override val id: String get() = ""
    override val label: CharSequence get() = app.getText(com.github.shadowsocks.R.string.plugin_disabled)
}
